gateways/authorizenet/authorizenet.go

```diff
@@ -159,6 +159,8 @@ func (client *AuthorizeNetClient) sendRequest(data Request) (*Response, error) {
 func getErrorCode(txnResponse TransactionResponse) string {
 	if len(txnResponse.Errors) > 0 {
 		return txnResponse.Errors[0].ErrorCode
+	} else if len(txnResponse.Messages) > 0 {
```

Comment:
don't understand why we would return the message code as error code.

Reply:
So, the message code can still contain error codes. Example, 311 indicates that you are trying to capture a transaction that has already been captured. This is in the message Code. While the transaction response error code will not have anything since the request itself was successful. 
This PR isn't super necessary, but it makes things nicer.